src/notifications/notificationmanager.cpp

```diff
@@ -414,7 +414,7 @@ NotificationList NotificationManager::GetNotifications(const QString &owner)
     QHash<uint, LipstickNotification *>::const_iterator it = notifications.constBegin(), end = notifications.constEnd();
     for ( ; it != end; ++it) {
         LipstickNotification *notification = it.value();
-        if (notification->owner() == owner) {
+        if (notification && notification->owner() == owner) {
```

Comment:
Wouldn't this be working around the problem, like would expect QHash not to ever contain null pointer values?